Documentation/component-description.md

```diff
@@ -59,7 +59,7 @@ In this page, we focus on the status of each elements. For requirements and desi
   - SNPE (stable)
   - Vivante (stable)
   - WIP: SNAP (Exynos-NPU & Qualcomm-SNPE), ...
-  - [Guide on writing a filter subplugin](Documentation/writing-subplugin-tensor-filter.md)
+  - [Guide on writing a filter subplugin](https://github.com/nnstreamer/nnstreamer/blob/main/Documentation/writing-subplugin-tensor-filter.md)
```

Comment:
Let's change the relative path to be compatible in the `github`.
After this PR, I will modify the script that generates `github.io` so that there is no problem in both `github` and `github.io`
Please check
 - $NNST_ROOT/*.md
 - $NNST_ROOT/Documentation/*.md
 - $NNST_ROOT/tools/*.md

```suggestion
  - [Guide on writing a filter subplugin](writing-subplugin-tensor-filter.md)
```